frontend/src/pages/main/home/client/components/finance/Finance.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import {Box, Typography} from "@mui/material";
+import {FinanceProps} from "./props";
+import {CURRENCY_USD} from "../../../../../../common/constansts/currency";
+import "./styles.css";
+
+const Finance: React.FC<FinanceProps> = ({income, expenses}) => {
+    return (
+        <Box>
+            <Typography
+                variant="h6"
+                className="finance"
+            >
+                Income: {income}{CURRENCY_USD}
+            </Typography>
+            <Typography
+                variant="h6"
+                className="finance"
+            >
+                Expenses: {expenses}{CURRENCY_USD}
+            </Typography>
```

Comment:
Look at your page. You have Title "Home", card "Balance, cards: "scan&pay" and so on, next card with "Transactions", income, expenses and companies in a legend. And font everywhere is different.

At least income, expenses and companies in a legend should be look same. Don't make it so big as h6. It's not header. There are plenty of other smaller sizes as `body1` https://mui.com/material-ui/react-typography/#usage

I would align it on the left side as well as legend.

I'm not sure why you like bold font, but it cannot be used everywhere, only for specific case.